tests/unit/test_es_manager.py

```diff
@@ -293,3 +345,15 @@ def test_constructLanguagePipeline(self, mocker):
         mockClient.put_pipeline.assert_called_once_with(
             id='testPipeline', body=testBody
         )
+
+    def test_splitWorkBatch_even(self):
+        firstHalf, secondHalf = ElasticsearchManager._splitWorkBatch([i for i in range(10)])
+
+        assert firstHalf == [i for i in range(5)]
+        assert secondHalf == [i for i in range(5, 10, 1)]
```

Comment:
I think you should delete the integer 1 out of the range since range() in Python increments by 1 by default.